src/reducers/set-checkout.js

```diff
@@ -167,12 +170,12 @@ export default function setCheckout (state = checkoutInitialState, action) {
         id: address.id,
         city: address.city,
         country_code: address.country,
-        email: address.meta_attributes.email.value,
+        email: t(address, 'meta_attributes.email.value').safeObject,
```

Comment:
Changes in this file fix a bug breaking checkout if someone started checking out with PayPal and then changed their mind and wanted to check out with Stripe. The addresses set by PayPal wouldn't have any meta_attributes and auto-populating the address form would crash. Typy was added here to fix this.